packages/ng-primitives/utils/src/forms/status.ts

```diff
@@ -32,20 +73,22 @@ export function controlStatus(): Signal<NgpControlStatus> {
 
   // Fallback if control is not yet available
   if (!control?.control) {
+    // There is still a chance that the control will be available i.e. after executing OnInit lifecycle hook
+    // in `formControlName` directive, so we set up an effect to subscribe to the control status
+    afterRenderEffect({
```

Comment:
Would it be more appropriate here to use afterNextRender as we are not looking to call this repeatedly any time a signal changes from what I can tell. I think in this case, we're just looking to run this after the initial render has completed.

Reply:
Yes, you are right, for some reason I thought `afterRenderEffect` would do the same at 4am 😄